README.md

```diff
@@ -1 +1,59 @@
-# Conference-Page
\ No newline at end of file
+## Screenshot of the project
+
+![screenshot2](https://user-images.githubusercontent.com/95374858/169492793-19a4f4ca-a9a1-449e-893f-8b1680995234.png)
+![screenshot1](https://user-images.githubusercontent.com/95374858/169492878-f272f7e2-4a1f-4c3a-a957-3f6eb3354e69.png)
+
+## SawaSawa Fashion Week.
+
+- This is a website for SawaSawa Fashion Week that entails the program of the days it will be held.
+
+## Link to loom video
+
+- To view a viedo with a decsription of the project you can click on this [link](https://www.loom.com/share/659ff5638a944135be6feddb0f400ee7).
+
+## Built With
+
+- HTML & CSS & Javascript.
+
+### Prerequisites
+
+- Git. Which is a open source distributed version control system designed for source code management
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+
+### Setup
+
+- Using git, clone the code to your machine or download a Zip File.
+
+## Authors
+
+👤 Grace Muthui
+
+- GitHub: [@Grayce Muthui](https://github.com/Graycemuthui)
+- Twitter: [@Grayce_Muthui](https://twitter.com/Grayce_Muthui)
+- LinkedIn: [@Grayce Muthui](http://www.linkedin.com/in/grayce-muthui-a17294226)
+
+## Deployment
+
+- To view a live demo of the project you can click on this [link](https://graycemuthui.github.io/Conference-Page/).
```

Comment:
- [ ] Please deploy the right branch of your portfolio project 👍🏿 currently deployed link is showing the readme 👍🏿